File: Warmup-1/missingChar/MissingChar.java
package missingChar;

public class MissingChar {
    //Retorne a String dada com o índice do caractere indicado removido.

    public String missingChar(String str, int n) {
        if (n > str.length()) {
          return "O número passado excede o valor máximo da String";
        } else {
            return (str.substring(0, n) + str.substring(n + 1));
        }
    }
}

File: Python/Warmup-1/missing_char.py
"""
Dado uma String "str", retorne uma nova onde será removido o caractere indicado por
"n", cujo valor será um válido (0..length-1, inclusive).
"""


def missing_char(str, n):
    return str[:n] + str[n+1:]


print(missing_char("hello", 2))
